dot/network/transaction_test.go

```diff
@@ -90,7 +72,5 @@ func TestHandleTransactionMessage(t *testing.T) {
 	}
 
 	s.handleTransactionMessage(peer.ID(""), msg)
-	for _, tx := range msg.Extrinsics {
-		require.NotNil(t, handler.txs[tx.Hash()])
-	}
+	mockhandler.AssertCalled(t, "HandleTransactionMessage", msg)
```

Comment:
does this check that `HandleTransactionMessage` was called with `msg`?

Reply:
Yes, the mock checks if the method was called with the exact arguments